gameplan/hooks.py

```diff
@@ -229,3 +229,5 @@
 # Recommended only for DocTypes which have limited documents with untranslated names
 # For example: Role, Gender, etc.
 # translated_search_doctypes = []
+
+search_index_path = '/Users/safwan/frappe-bench/apps/test_index'
```

Comment:
can't have hardcoded paths

Reply:
What do you suggest we do - leave it for individual users to write it in `hooks.py`, or calculate and store in that site/GP app?

Reply:
Try `frappe.get_site_path('indexes', 'apex_search')`